core/src/main/java/edu/wpi/grip/core/exception/GripException.java

```diff
@@ -0,0 +1,26 @@
+
+package edu.wpi.grip.core.exception;
+
+/**
+ * An exception thrown when something goes wrong with an internal GRIP
+ * operation.
+ */
+public class GripException extends RuntimeException {
+
+    public GripException() {
+        super();
+    }
+
+    public GripException(String message) {
+        super(message);
+    }
+
+    public GripException(Throwable cause) {
```

Comment:
Remove this constructor.
If we're going to add a `GripException` we should require that a message is provided.